ui/src/app/edge/live/flat/flat-widget-line/abstract-flat-widget-line.ts

```diff
@@ -28,6 +28,10 @@ export abstract class AbstractFlatWidgetLine implements OnDestroy {
      */
     private selector: string = UUID.UUID().toString();
 
+    /** 
+     * displayName is the displayed @Input value in html
+     */
+    public displayName: string;
```

Comment:
Wofür brauchst du den `displayName`?

Reply:
hab ich gemacht für this.isInitialized. Wenn nur ein name übergeben wird und kein channel oder value dann wird der name nicht angezeigt

Reply:
fällt jetzt raus